PraceDomowe/PracaDomowa5/Dwuznik Mariusz Praca Domowa 5/PD5_Dwuznik.ipynb

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
<p>Prawie dobrze, jedyny błąd polega na tym że liczba pytań w zbiorze to nie są tylko te Posts dla których istnieje AcceptedAnswer, tylko te, których PostTypeId = 1 (chociaż to trzeba by było odczytać z opisu danych).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>